cinema/tests/test_cinema_hall_api.py

```diff
@@ -43,9 +41,7 @@
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.data[1]["name"], vip_hall["name"])
         self.assertEqual(response.data[1]["rows"], vip_hall["rows"])
-        self.assertEqual(
-            response.data[1]["seats_in_row"], vip_hall["seats_in_row"]
-        )
+        self.assertEqual(response.data[1]["seats_in_row"], vip_hall["seats_in_row"])
```

Comment:
The repeated assertion of `response.status_code` in `test_get_cinema_halls` is unnecessary. You can remove the second assertion on line 41.